packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -190,7 +190,18 @@ Future<void> _runBuild(
 
   // MSBuild sends all output to stdout, including build errors. This surfaces
   // known error patterns.
-  final RegExp errorMatcher = RegExp(r':\s*(?:warning|(?:fatal )?error).*?:');
+  final List<String> errorMessages = <String>[
+    r':\s*(?:warning|(?:fatal )?error).*?:',
+    r'Error detected in pubspec.yaml:',
```

Comment:
Period should be escaped to not match any character:

```suggestion
    r'Error detected in pubspec\.yaml:',
```